frontend/src/concepts/roleBinding/RoleBindingPermissionsTable.tsx

```diff
@@ -78,16 +125,11 @@ const RoleBindingPermissionsTable: React.FC<RoleBindingPermissionsTableProps> =
                 labels,
                 ownerReference,
               );
-              createRoleBinding(newRBObject)
-                .then(() => {
-                  onDismissNewRow();
-                  refresh();
-                })
-                .catch((e) => {
-                  onError(e);
-                });
+              createProjectRoleBinding(subjectName, newRBObject, newRBObject);
             }}
             onCancel={onDismissNewRow}
+            onDelete={() => null}
+            onEdit={() => null}
```

Comment:
You could always make these optional on the component so you don't have to pass these props